build.xml

```diff
@@ -0,0 +1,310 @@
+        </SymfonyConsole>
+    </target>
+
+    <target name="symfony-assets-install-dev" description="Install assets in dev mode">
+        <SymfonyConsole command="assets:install" checkreturn="true" console="${symfony-bin-console}">
+            <arg name="env" value="dev" />
+            <arg value="--no-interaction" />
+        </SymfonyConsole>
+    </target>
+
+    <target name="symfony-assets-install-prod" description="Install assets in prod mode">
+        <SymfonyConsole command="assets:install" checkreturn="true" console="${symfony-bin-console}">
+            <arg name="env" value="prod" />
+            <arg value="--no-debug" />
+            <arg value="--no-interaction" />
+        </SymfonyConsole>
+    </target>
+
+    <target name="symfony-assetic-dump-dev" description="Dumps all assets to the filesystem for dev environment">
+        <SymfonyConsole command="assetic:dump" checkreturn="true" console="${symfony-bin-console}">
+            <arg name="env" value="dev" />
+            <arg value="--no-interaction" />
+        </SymfonyConsole>
+    </target>
+
+    <target name="symfony-assetic-dump-prod" description="Dumps all assets to the filesystem for prod environment">
+        <SymfonyConsole command="assetic:dump" checkreturn="true" console="${symfony-bin-console}">
+            <arg name="env" value="prod" />
+            <arg value="--no-debug" />
+            <arg value="--no-interaction" />
+        </SymfonyConsole>
+    </target>
+
+    <target name="symfony-doctrine-migrations-migrate" description="Executes migrations">
+        <if>
+            <istrue value="${project.symfony-doctrine-migrations-enable}"/>
+            <then>
+                <SymfonyConsole command="doctrine:migrations:migrate" checkreturn="true" console="${symfony-bin-console}">
+                    <arg name="no-interaction"/>
+                </SymfonyConsole>
+            </then>
+        </if>
+    </target>
+
+    <target name="symfony-doctrine-schema-validate" description="Validates the doctrine mapping files"
+            depends="symfony-doctrine-migrations-migrate">
+        <SymfonyConsole command="doctrine:schema:validate" checkreturn="true" console="${symfony-bin-console}">
+            <arg name="no-interaction"/>
+        </SymfonyConsole>
+    </target>
+
+    <target name="initialize-autoload" description="Initialize autoloader">
+        <autoloader />
+    </target>
+
+    <target name="phpmd" description="Run phpmd checks" depends="initialize-autoload">
+        <phpmd rulesets="${project-file-config-phpmd}">
+            <fileset refid="src-php"/>
+        </phpmd>
+    </target>
+
+    <target name="phpcpd" description="Find duplicates in code" depends="initialize-autoload">
+        <phpcpd>
+            <fileset refid="src-php"/>
+            <formatter type="default" useFile="false"/>
+        </phpcpd>
+    </target>
+
+    <target name="phpcs" description="Run code style checks" depends="initialize-autoload">
```

Comment:
Unfortunately phing has issue when we don't want to save report to file, so we have to save report to file and than output it